### ISSUE TITLE
Return user to same Facility Locator search results state following refresh

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#30919]
## Issue Description
Following the return of pagination to Facility Locator search results, we need to update the expected behavior following a page refresh. 

## Tasks
 Update behavior when page is refreshed

## Acceptance Criteria
-[ ]  When the Facility Locator page is refreshed, users should be returned to the same search results state

**Comment by mmiddaugh** [comment]
Behavior is still present
